src/loaders/LoaderUtils.js

```diff
@@ -12,6 +12,12 @@ var LoaderUtils = {
 
 		}
 
+		if ( typeof Buffer !== 'undefined' ) {
+
+			return new Buffer( array ).toString( 'ascii' );
+
+		}
```

Comment:
This method is otherwise expected to allow utf-8 (for example, nodes in many formats have human-readable names that might use non-ASCII characters. Does using the default UTF8 encoding from `toString()` cause any issues? If so perhaps a second argument to `decodeText( array, encoding )` would be appropriate — TextDecoder would also support that.

Reply:
utf8 should be fine, though I'm not 100% sure whether it's possible to slip non-ASCII non-UTF8 bits into the model name in the header or what happens if the decoding fails at that stage...

Reply:
STL ASCII spec says "ASCII" and mentions no alternate encodings, but you never know what you're gonna get. ;) *Should* be utf8-safe.  http://www.fabbers.com/tech/STL_Format#Sct_ASCII